base/src/test/user_model/test_set_functions_error_handling.rs

```diff
@@ -0,0 +1,93 @@
+use crate::test::util::new_empty_model;
```

Comment:
This is the wrong place for this file because it doesn't deal with the user model interface

Reply:
Removed from there